server/web/router_test.go

```diff
@@ -1097,3 +1098,22 @@ func TestRouterAddRouterPointerMethodPanicNotImplementInterface(t *testing.T) {
 	handler := NewControllerRegister()
 	handler.AddRouterMethod(method, "/user", (*TestControllerWithInterface).PingPointer)
 }
+
+func TestGetAllControllerInfo(t *testing.T) {
```

Comment:
you still need to add more test cases to cover `composeControllerInfos` all path